api/src/main/java/org/openmrs/api/db/hibernate/HibernateCohortDAO.java

```diff
@@ -51,26 +53,42 @@ public void setSessionFactory(SessionFactory sessionFactory) {
 	public Cohort getCohort(Integer id) throws DAOException {
 		return (Cohort) sessionFactory.getCurrentSession().get(Cohort.class, id);
 	}
-	
+
+	/**
+	 * @see org.openmrs.api.db.CohortDAO#getCohortsContainingPatientId(java.lang.Integer)
+	 */
+	@SuppressWarnings("unchecked")
+	public List<Cohort> getCohortsContainingPatientId(Integer patientId, Boolean voided) throws DAOException {
+		Disjunction orDisjunction = Restrictions.disjunction();
+		orDisjunction.add(Restrictions.isNull("m.endDate"));
+		orDisjunction.add(Restrictions.gt("m.endDate", new Date()));
+		return (List<Cohort>) sessionFactory.getCurrentSession().createCriteria(Cohort.class)
+				.add(Restrictions.eq("voided", voided))
+				.createAlias("members", "m")
+				.add(Restrictions.le("m.startDate", new Date()))
+				.add(orDisjunction)
+				.createAlias("m.patient", "p")
+				.add(Restrictions.eq("p.patientId", patientId))
```

Comment:
So I replaced those two lines with:
				`.add(Restrictions.eq("m.patient", new Patient(patientId)))`
Yet it doesn't seem to work. Won't "m.patient" and new Patient(patientId) be two different objects?